src/guide/working-with-pull-requests.md

```diff
@@ -12,7 +12,7 @@ This section also assumes that you have already read [I have a patch, what do I
 
 ## Think once more
 
-All code reviews in OpenJDK are done in public. Once you open your PR for public review the internet can see it and comment on it. Make sure your code is ready for it. Look through your comments, make sure that temporary code is gone, and make sure you have sanitized your method and variable names.
+All code reviews in OpenJDK are done in public. Once you open your PR for public review the internet can see it and comment on it. Make sure your code is ready for it. Look through your comments, make sure that temporary code is gone, and make sure you have sanitized your method and variable names. Also make sure you understand your code. Why is it working? What are the potential pitfalls? What are the edge-cases? If you haven't already in the mail conversation that preceeded this PR, it's likely that you will need to answer all these questions during the review.
```

Comment:
```suggestion
All code reviews in OpenJDK are done in public. Once you open your PR for public review the internet can see it and comment on it. Make sure your code is ready for it. Look through your comments, make sure that temporary code is gone, and make sure you have sanitized your method and variable names. Also, make sure you understand your code. Why is it working? What are the potential pitfalls? What are the edge-cases? If you haven't already answered all these questions in the mail conversation that preceded this PR, it's likely that you will need to answer them during the review.
```
It's better not to omit the verb and its object in the conditional sentence that goes before the main one, the reader doesn't know yet about the things that are omitted.

Reply:
Agreed. Thanks!